OpenRA.Mods.Common/Traits/Conditions/GrantChargedConditionOnToggle.cs

```diff
@@ -35,38 +35,47 @@ public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo
 		[Desc("Cooldown (in ticks) to reach full charge.")]
 		public readonly int ChargeDuration = 500;
 
-		[Desc("The amount of charge that needs to be present for deploy to be issued. " +
+		[Desc("The amount of charge that needs to be present to turn on the condition. " +
 			"If set to -1, threshold is set to full charge. " +
 			"If activated without full charge " + nameof(ConditionDuration) + " is percentally smaller.")]
 		public readonly int ChargeThreshhold = -1;
 
 		[Desc("How long (in ticks) should the condition stay active?")]
 		public readonly int ConditionDuration = 1;
 
-		[Desc("Can " + nameof(DeployedCondition) + " be canceled by followup deploy order?")]
+		[Desc("Can " + nameof(ActivatedCondition) + " be canceled turned off manually?")]
 		public readonly bool CanCancelCondition = false;
 
+		[Desc("Should we interrupt the current activity")]
```

Comment:
"... if not queued" is an important bit of information here.

Reply:
it makes it more confusing, in the same way my addition was